app/templates/cluster_viz_chartjs.html

```diff
@@ -396,13 +396,13 @@ <h5 class="modal-title" id="exampleModalLongTitle">Clustering in progress</h5>
                     <p style="text-align: left;">Filter by date</p>
                     <label for="filter-from-date">From</label>
                     <select class="filter-from-date" id="filter-from-date">
-                        {% for date in columns[date_column] %}
+                        {% for date in columns[date_column]|sort %}
                             <option value="{{ date }}">{{ date}}</option>
                         {% endfor %}
                     </select>
                     <label for="filter-to-date">to</label>
                     <select class="filter-to-date" id="filter-to-date">
-                        {% for date in columns[date_column] %}
+                        {% for date in columns[date_column]|sort(reverse=true) %}
```

Comment:
sorted(columns[date_column]) does not work?

Reply:
This is syntax for dict in jinja template, not always equals to python. See docs for more:

https://jinja.palletsprojects.com/en/3.1.x/templates/